### ISSUE TITLE
Support Bulleted List, Ordered List.

### ISSUE THREAD

**Issue by chuyenphamdev** [canopas/rich-editor-swiftui#72]
Hi,
Do you have any plans to support Bulleted Lists and Ordered Lists in the future?

Thanks.

**Comment by cp-divyesh-v** [comment]
@chuyenphamdev Yes, I am not able to promise exact date/time, But we have it in our future scope.
